apps/service_providers/llm_service/runnables.py

```diff
@@ -370,11 +365,11 @@ def _get_file_name_and_link_for_citation(self, file_id: str, forbidden_file_ids:
 
         if file_id not in forbidden_file_ids:
             # Don't allow downloading assistant level files
-            team_slug = self.state.session.team.slug
+            team = self.state.session.team
             session_id = self.state.session.id
             try:
-                file = File.objects.get(external_id=file_id, team__slug=team_slug)
-                file_link = f"file:{team_slug}:{session_id}:{file.id}"
+                file = File.objects.get(external_id=file_id, team_id=team.id)
```

Comment:
just FYI: `team_id=team.id` in a query filter is equivalent to `team=team` (Django does the ID part behind the scenes)

Reply:
Got it, thanks